### PR TITLE
Php8 compability

### PR DESCRIPTION
**Pull request by fritz-gerneth**

### CODE REVIEW

**Comment by fritz-gerneth** [comment]
This should resolve once the `service-bus` dependency has been updated

**Comment by fritz-gerneth** [comment]
@prolic you know if there is any way to rerun travis without a new commit?

**Comment by prolic** [comment]
If you log in, there should be a button, at least there was one in the past.

**Comment by fritz-gerneth** [comment]
Ah it is there, but disabled for me - I assume as this is not my repository. Can you trigger a re-test eventually?

**Comment by prolic** [comment]
Sure, later today.

File: composer.json [hunk 1]
```diff
@@ -22,16 +22,17 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.1",
+        "php": "^7.3 | ^8.0",
         "prooph/common" : "^4.2.1",
         "prooph/service-bus" : "^6.2",
```
Comment (prolic) [inline composer.json]: require 6.3 now

**Review by prolic — CHANGES_REQUESTED** [review]
Please update the year in LICENSE file as well.

**Comment by prolic** [comment]
https://github.com/prooph/event-store-bus-bridge/releases/tag/v3.4.0

**Pull request merged**